lgc/patch/NggPrimShader.cpp

```diff
@@ -1196,66 +1221,72 @@ void NggPrimShader::constructPrimShaderWithoutGs(Module *module) {
     vertCountInSubgroup = m_builder->CreateIntrinsic(Intrinsic::amdgcn_readlane, {},
                                                      {vertCountInWaves, m_builder->getInt32(waveCountInSubgroup)});
 
-    // Get vertex count for all waves prior to this wave
-    vertCountInPrevWaves =
-        m_builder->CreateIntrinsic(Intrinsic::amdgcn_readlane, {}, {vertCountInWaves, m_nggFactor.waveIdInSubgroup});
+    if (disableCompact)
+      m_builder->CreateBr(endCompactVertBlock);
+    else {
```

Comment:
Please put braces around the `if` block because there are braces for the `else`.